Do-It/Controller/DoItTableViewController.swift

```diff
@@ -85,6 +113,7 @@ extension DoItTableViewController {
         cell.courseLabel?.text = doIt.course.name
         cell.dateLabel?.text = formattedDate(doIt.dueDate)
         cell.descriptionLabel?.text = doIt.description
+        cell.checkmarkImageView.isHidden = interactionMode != .selectingToShare || !selectedIndexPaths.contains(indexPath)
```

Comment:
Readability Nit: For the `checkmarkImageView` to be shown both predicates have to be false. This can be confusing because there are negatives in each predicate as well. Consider wrapping this logic in a function or converting this using DeMorgan's Law (  `!(interactionMode == .selectingToShare) && selectedIndexPaths.contains(indexPath)` )